cmd/containerd-shim-runhcs-v1/task.go

```diff
@@ -117,7 +116,6 @@ func isStatsNotFound(err error) bool {
 	return errdefs.IsNotFound(err) ||
 		hcs.IsNotExist(err) ||
 		hcs.IsOperationInvalidState(err) ||
-		gcs.IsNotExist(err) ||
```

Comment:
This just because hcs.IsNotExist and the errno the gcs version were checking are the exact same?

Reply:
Yup, the `gcs` version was checking for the result being `0xc037010e`, but since we can now unwrap those errors, `hcs.IsNotExist(` will unwrap in `errors.Is(` and check against `ErrComputeSystemDoesNotExist = syscall.Errno(0xc037010e)`